### PR TITLE
Loading a Database Dump in LOCAL environment

### PR DESCRIPTION
**Pull request by jmborr**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -21,14 +21,17 @@ all: wheel/dasmon wheel/webmon wheel/workflow SNSdata.tar.gz
...
 
 create/mamba:  ## create conda environment "webmon" with file conda_environment.yml using mamba
 	conda env create --name webmon python=$(PYTHON_VERSION)
 	conda install --name base -c conda-forge mamba
 	mamba env update --name webmon --file conda_environment.yml
+	mamba env update --name webmon --file conda_development.yml
 
 docker/pruneall: ## stop and force remove all containers, images and volumes
 	docker system prune --force --all --volumes
+	docker system prune --force --all --volumes
```
Comment (rosswhitfield) [inline Makefile:34]: repeated line?

File: docker-compose.yml [hunk 2]
```diff
@@ -21,18 +21,21 @@ services:
       context: .
       dockerfile: Dockerfile.webmon
       # build using the host's network, required in certain environments when accessing the debian repositories
+      network: host
```
Comment (rosswhitfield) [inline docker-compose.yml:24]: Why?

File: docs/developer/instruction/deployment.rst [hunk 3]
```diff
@@ -32,6 +32,7 @@ LDAP_CERT_FILE         yes    `ldap.OPT_X_TLS_CACERTFILE <https://www.python-lda
 LDAP_DOMAIN_COMPONENT  yes    Use in `AUTH_LDAP_USER_DN_TEMPLATE <https://django-auth-ldap.readthedocs.io/en/latest/reference.html#std:setting-AUTH_LDAP_USER_DN_TEMPLATE>`_
 LDAP_SERVER_URI        yes    `AUTH_LDAP_SERVER_URI <https://django-auth-ldap.readthedocs.io/en/latest/reference.html#auth-ldap-server-uri>`_
 TIME_ZONE                     `Time zone to use <https://docs.djangoproject.com/en/3.2/ref/settings/#time-zone-1>`_
+LOAD_INITIAL_DATA             In non-production environments, loads default database "db_init.json" when set to "true". Set to "false" if loading a database dump instead
```
Comment (rosswhitfield) [inline docs/developer/instruction/deployment.rst:35]: `.json`?

File: docs/developer/instruction/docker.rst [hunk 4]
```diff
@@ -21,7 +21,7 @@ An additional flag ``-d`` can be supplied to run docker in detached mode.
    Use ``docker-compose --file <filename>`` to select a different configuration
 
 To start a single image, supply its name as an additional argument to ``docker-compose up``.
-
+wait-for-it.sh
```
Comment (rosswhitfield) [inline docs/developer/instruction/docker.rst:24]: ?

**Review by rosswhitfield — CHANGES_REQUESTED** [review]
You mentioned moving everything out of the `.env` file into the docker-compose.yml, did you want to do that as part of this PR? :smile:

File: docker-compose.yml (hunk 2 above)
Comment (jmborr) [reply docker-compose.yml:24]: I build the apps in `ndav4.sns.gov` and docker cannot "reach out" to remote package repositories when creating the images. This is the only time when the host network is used, once the images are created the host network is not used

File: Makefile (hunk 1 above)
Comment (jmborr) [reply Makefile:34]: I think I didn't solve the conflicts in the `Makefile` as good as I thought :facepalm: , thanks for catching this

File: docs/developer/instruction/docker.rst (hunk 4 above)
Comment (jmborr) [reply docs/developer/instruction/docker.rst:24]: got rid of it

**Review by rosswhitfield — APPROVED** [review]

**Comment by jmborr** [comment]
> You mentioned moving everything out of the `.env` file into the docker-compose.yml, did you want to do that as part of this PR? smile

I did give it a try but came to believe it's not a good practice because we use many variables (especially the `DATABASE_*` ones) in many containers and their names and values need to the same. Just as duplicate code is a bad practice, duplicating initialization of variables is a bad practice too.

Getting rid of .env worked well in `web reflectivity` because we don't have duplication of variables among containers

**Pull request merged**